src/components/pages/parameter-or-variable.jsx

```diff
@@ -20,15 +21,39 @@ const ParameterOrVariablePage = React.createClass({
     currency: PropTypes.string.isRequired,
     location: locationShape.isRequired,
     params: PropTypes.shape({ name: PropTypes.string.isRequired }).isRequired, // URL params
-    parameters: PropTypes.arrayOf(AppPropTypes.parameterOrScale).isRequired,
+    parameters: PropTypes.objectOf(AppPropTypes.parameterOrScale).isRequired,
     variables: PropTypes.arrayOf(AppPropTypes.variable).isRequired,
   },
+  getInitialState() {
+    const variable = this.props.variables.find(variable => variable.name === this.props.params.name)
+    return {variable: variable, waitingForResponse: true}
```

Comment:
> **META**
>
> In order to avoid frustration for all contributors, I would recommend to be more explicit about how important something is.
> 
> If it's only a matter of personal preference, then don't make your review sound as an imperative change in order to allow the merge. This can be done either through phrasing (“I'd feel more at ease with…”, “I'd prefer…”, “I'd like…”) or through quote markup.
> 
> If you really feel that this name is harmful, or is a large missed opportunity, then a reference would be welcome. I personally don't believe that `loading` is so much more common than `waitingForResponse`, for example :wink: